docs/apis/core/dml/delegated-transactions.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Delegated database transactions
```

Comment:
It might be better to simplify this to 'Transactions', at least in the sidebar where it's already clear that the topic is a part of the DML API.

If you want the page title to be "Delegated database transactions" still, you can use the `sidebar_label` YAML entry to support this:

```suggestion
title: Delegated database transactions
sidebar_label: Transactions
```

Reply:
I've just changed the main title with out adding the sidebar label. Happy to change this to a side bar label if that is preferred.